vhdbuilder/parts/k8s/cloud-init/artifacts/kubesystem.slice

```diff
@@ -0,0 +1,7 @@
+[Unit]
```

Comment:
@adilliadil , let's make a change to consolidate this vhdbuilder folder with the parts folder. We should haven't done this when we move the vhd builder from aks rp to here.

Reply:
True, I will prioritize this task